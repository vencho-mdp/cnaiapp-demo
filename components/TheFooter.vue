<template>
  <footer
    class="bg-primary-darkblue flex mt-2 w-full p-2 items-center justify-between"
  >
    <button
      data-test="bug_report_btn"
      style="max-width: 200px"
      class="bg-primary-blue rounded font-bold shadow-sm text-xs text-center py-1 px-2 w-1/3 hover-effect !text-white"
      @click="$store.commit('toggle_bug_reporting_sidebar')"
    >
      Comunicar un error
    </button>
    <span class="flex font-p font-bold text-xs pl-4 !text-white">
      Hecho por: &nbsp;
      <a
        href="https://www.linkedin.com/in/benicio-cardozo-115421219/"
        class="underline"
        target="_blank"
      >
        Benicio Cardozo</a
      >
    </span>
  </footer>
</template>
