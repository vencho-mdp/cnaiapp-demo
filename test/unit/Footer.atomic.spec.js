import { shallowMount } from '@vue/test-utils'
import TheFooter from '@/components/TheFooter.vue'

describe('VTitle', () => {
  it('should render correctly', async () => {
    const el = shallowMount(TheFooter)
    const html = el.html()
    expect(html).toMatchSnapshot()
  })
})
