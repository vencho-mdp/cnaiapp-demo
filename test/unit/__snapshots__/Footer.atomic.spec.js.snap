// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VTitle should render correctly 1`] = `
"<footer class=\\"bg-primary-darkblue flex mt-2 w-full p-2 items-center justify-between\\"><button data-test=\\"bug_report_btn\\" class=\\"bg-primary-blue rounded font-bold shadow-sm text-xs text-center py-1 px-2 w-1/3 hover-effect !text-white\\" style=\\"max-width: 200px;\\">
    Comunicar un error
  </button> <span class=\\"flex font-p font-bold text-xs pl-4 !text-white\\">
    Hecho por: &nbsp;
    <a href=\\"mailto:beniciocardozomdp@gmail.com\\" class=\\"underline\\">
      Benicio Cardozo</a></span></footer>"
`;
